snscrape/modules/telegram.py

```diff
@@ -240,20 +213,20 @@ def get_items(self):
 		nextPageUrl = ''
 		while True:
 			yield from self._soup_to_items(soup, r.url)
-			try:
-				if soup.find('a', attrs = {'class': 'tgme_widget_message_date'}, href = True)['href'].split('/')[-1] == '1':
+			dateElt = soup.find('a', attrs = {'class': 'tgme_widget_message_date'}, href = True)
+			if dateElt and 'href' in dateElt.attrs:
```

Comment:
This check resolves the `try-except` complaints mentioned in https://github.com/JustAnotherArchivist/snscrape/pull/413#discussion_r1052751801